internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestBatch.java

```diff
@@ -815,6 +817,7 @@ private String testBatchStore(
 
     String inputDirPath = "file://" + inputDir.getAbsolutePath();
     Properties props = defaultVPJProps(veniceCluster, inputDirPath, storeName);
```

Comment:
I wonder if we should run these test with compute engines?

Reply:
You mean with both engines? I believe it will double most of our CI times since VPJ and the verification after that is the part that takes the longest.

I only updated TestBatch and TestVsonStoreBatch to run with Spark, but other tests will still run with MapReduce - so, we should still have good enough coverage, IMO

Reply:
sounds good